css/templates/e-commerce/page.css

```diff
@@ -0,0 +1,216 @@
+@import '../../core';
+@import 'page-vars';
+@import '../../../components/inputs/inputs';
+@import '../../../components/social-follow/social-follow';
+
+body {
+  font-size: var(--narrow-body-font-size);
+}
+
+@media (--breakpoint-md) {
+  body {
+    font-size: var(--wide-body-font-size);
+  }
+}
+
+main {
+  max-width: none;
+}
+
+
+/* TYPOGRAPHY */
+.h3 {
+  color: var(--color-font-title);
+  line-height: normal;
+  padding-bottom: var(--space-1);
+}
+
+
+/* ELEMENTS */
+.ampstart-btn-secondary {
+  background-color: var(--color-white);
+  color: var(--color-primary);
+  border-color: var(--color-primary);
+  font-size: var(--button-font-size);
+  font-family: var(--font-family);
+  font-weight: var(--bold-font-weight);
+}
+
+.ampstart-hero .ampstart-btn {
+  padding: var(--space-1) var(--space-3);
+}
+
+
+/* HEADER */
+.ampstart-headerbar {
+  color: var(--color-primary);
+  background-color: var(--color-white);
+  height: var(--narrow-nav-height);
+  z-index: var(--z1);
+}
+
+.ampstart-navbar-trigger {
+  cursor: pointer;
+  height: 24px;
+  color: var(--color-primary);
+}
+
+@media (--breakpoint-md) {
+  .ampstart-headerbar {
+    height: var(--wide-nav-height);
+  }
+
+  .ampstart-headerbar .ampstart-navbar-trigger {
+    padding: 0;
+    margin-left: var(--space-1);
+  }
+}
+
+.ampstart-headerbar-title {
+  font-size: var(--h4);
+  font-weight: var(--bold-font-weight);
+  line-height: normal;
+  color: var(--color-secondary);
+}
+
+@media (--breakpoint-md) {
+  .ampstart-headerbar-title {
+    font-size: var(--h3);
+  }
+}
+
+.ampstart-headerbar-home-link {
+  padding-bottom: 0;
+  text-indent: -var(--space-2);
+  color: var(--color-primary);
+}
+
+.ampstart-sidebar {
+  background-color: var(--color-white);
+  width: var(--sidebar-width);
+  margin-bottom: var(--space-3);
+}
+
+.ampstart-sidebar-header {
+  position: relative;
+  z-index: var(--z1);
+}
+
+.ampstart-sidebar .ampstart-navbar-trigger {
+  margin-top: var(--space-3);
+  font-size: var(--space-3);
+  color: var(--color-primary);
+}
+
+.ampstart-nav {
+  margin: var(--space-3) 0 var(--space-4) 0;
+}
+
+.ampstart-nav-item {
+  color: var(--color-primary);
+}
+
+.ampstart-nav-link {
+  font-size: var(--h3);
+  font-weight: var(--bold-font-weight);
+  line-height: normal;
+  display: block;
+  margin-bottom: var(--space-3);
+}
+
+.ampstart-sidebar-faq {
+  width: 100%;
+  color: var(--color-secondary);
+  font-family: var(--font-family-secondary);
+  padding-top: var(--space-2);
+}
+
+.ampstart-faq-item {
+  line-height: normal;
+  padding: 0 0 var(--space-1) 0;
+}
+
+.ampstart-social-follow {
+  justify-content: initial;
+  display: inline-block;
+}
+
+.ampstart-social-follow li {
+  display: inline-block;
+}
+
+.ampstart-social-follow li:last-child {
+  margin-right: 0;
+}
+
+.main {
+  margin-top: var(--narrow-nav-height);
+}
+
+@media (--breakpoint-md) {
+  .main {
+    margin-top: calc(var(--wide-nav-height));
+  }
+}
+
+
+/* HERO */
+.e-commerce-ampstart-hero-content {
```

Comment:
Changed to commerce-hero-content although this doesn't feel right. Can be easily renamed when an appropriate name has been found.